datasets/cloud_storage_geo_index/pipelines/cloud_storage_geo_index/cloud_storage_geo_index_dag.py

```diff
@@ -0,0 +1,135 @@
+    "depends_on_past": False,
+    "start_date": "2021-03-01",
+}
+
+
+with DAG(
+    dag_id="cloud_storage_geo_index.cloud_storage_geo_index",
+    default_args=default_args,
+    max_active_runs=1,
+    schedule_interval="0 1 0 0 6",
+    catchup=False,
+    default_view="graph",
+) as dag:
+    create_cluster = kubernetes_engine.GKECreateClusterOperator(
+        task_id="create_cluster",
+        project_id="{{ var.value.gcp_project }}",
+        location="us-central1-c",
+        body={
+            "name": "cloud-storage-geo-index",
+            "initial_node_count": 4,
+            "network": "{{ var.value.vpc_network }}",
+            "node_config": {
+                "machine_type": "e2-standard-16",
+                "oauth_scopes": [
+                    "https://www.googleapis.com/auth/devstorage.read_write",
+                    "https://www.googleapis.com/auth/cloud-platform",
+                ],
+            },
+        },
+    )
+
+    # Run Landsat index transform within kubernetes pod
+    landsat_index = kubernetes_engine.GKEStartPodOperator(
+        task_id="landsat_index",
+        name="landsat_index",
+        project_id="{{ var.value.gcp_project }}",
+        location="us-central1-c",
+        cluster_name="cloud-storage-geo-index",
+        namespace="default",
+        image_pull_policy="Always",
+        image="{{ var.json.cloud_storage_geo_index.container_registry.run_csv_transform_kub }}",
+        env_vars={
+            "PIPELINE_NAME": "{{ var.json.cloud_storage_geo_index.landsat_index.pipeline_name }}",
+            "SOURCE_URL": "{{ var.json.cloud_storage_geo_index.landsat_index.source_url }}",
+            "SOURCE_ZIPFILE": "{{ var.json.cloud_storage_geo_index.landsat_index.source_zipfile }}",
+            "SOURCE_FILE": "{{ var.json.cloud_storage_geo_index.landsat_index.source_file }}",
+            "TARGET_FILE": "{{ var.json.cloud_storage_geo_index.landsat_index.target_file }}",
+            "CHUNKSIZE": "{{ var.json.cloud_storage_geo_index.landsat_index.chunksize }}",
+            "PROJECT_ID": "{{ var.value.gcp_project }}",
+            "DATASET_ID": "{{ var.json.cloud_storage_geo_index.dataset_id }}",
+            "TABLE_ID": "{{ var.json.cloud_storage_geo_index.landsat_index.table_id }}",
+            "TARGET_GCS_BUCKET": "{{ var.value.composer_bucket }}",
+            "TARGET_GCS_PATH": "{{ var.json.cloud_storage_geo_index.landsat_index.target_gcs_path }}",
+            "SCHEMA_PATH": "{{ var.json.cloud_storage_geo_index.landsat_index.schema_path }}",
+            "DROP_DEST_TABLE": "{{ var.json.cloud_storage_geo_index.landsat_index.drop_dest_table }}",
+            "INPUT_FIELD_DELIMITER": "{{ var.json.cloud_storage_geo_index.landsat_index.input_field_delimiter }}",
+            "REMOVE_SOURCE_FILE": "{{ var.json.cloud_storage_geo_index.landsat_index.remove_source_file }}",
+            "DELETE_TARGET_FILE": "{{ var.json.cloud_storage_geo_index.landsat_index.delete_target_file }}",
```

Comment:
I think we're over parameterizing here. We should make most of these variables transparent, because they're non-sensitive variables. The only variables we need to parameterize are `PROJECT_ID` and `TARGET_GCS_BUCKET` because they **do** vary per project. The dataset and table names, schema paths, filenames, delimiters are constants in the process.

Reply:
Good idea, but when they do change (e.g. source url, source_file, chunksize) we can just modify the values in the variables and we are good.  That's my take anyway.

Reply:
That's alright. The goal is for us to show everyone (because our code is open source) how we set these values for ourselves, so people reading our code can have a more accuracte sense of how things are configured based on actual values.

Reply:
Regardless... Done.